dll/win32/shell32/folders/CRecycleBin.cpp

```diff
@@ -1018,6 +1018,65 @@ TRASH_TrashFile(LPCWSTR wszPath)
     return DeleteFileToRecycleBin(wszPath);
 }
 
+static void TRASH_PlayEmptyRecycleBinSound()
+{
+    CRegKey regKey;
+    CHeapPtr<TCHAR> pszValue;
```

Comment:
please use WCHAR, not TCHAR.
TCHAR is a compatibility thing if you need to share code with Win98 or the likes.

Reply:
`CRegKey` still uses `TCHAR`.

Reply:
It would use automatically WCHAR if shell32 is compiled in unicode (which should be specified in its cmakefile)

Reply:
CRegKey is written to accept both WCHAR and CHAR, because it needs to be compatible with outdated and / or user programs.